File: server/features/unfinished_work.go
package features

import "github.com/tstapler/stapler-squad/server/featureregistry"

// UnfinishedWork describes the unfinished-work RPCs and their UI surface.
var UnfinishedWork = featureregistry.Feature{
	ID:          "unfinished-work",
	Title:       "Unfinished Work",
	Description: "Surfaces pending changes across git worktrees.",
	RPCIDs:      []string{"unfinished:list", "unfinished:watch"},
	Status:      featureregistry.StatusStable,
	Since:       "1.0.0",
}

func init() {
	featureregistry.Register(UnfinishedWork)
}

File: web-app/src/lib/features/features/unfinished-work.ts
import type { Feature } from '../types';

export const unfinishedWorkFeatures = {
  'unfinished-work': {
    id: 'unfinished-work',
    title: 'Up Next',
    description: 'Surfaces pending changes across git worktrees, open PRs, and queued backlog items.',
    rpcIds: ['unfinished:list', 'unfinished:watch'],
    componentPaths: [
      'app/unfinished/UnfinishedTab.tsx',
      'components/unfinished/BacklogQueueSection.tsx',
    ],
    testIds: ['Unfinished Work > shows unfinished work panel'],
    status: 'stable',
    since: '1.0.0',
  },
} as const satisfies Record<string, Feature>;
